.github/workflows/commit_artifacts.yml

```diff
@@ -329,7 +329,7 @@ jobs:
           git status
           echo "===================="
           echo "Checking for changes"
-          if git status --porcelain | grep -qv '/REVISION'; then
+          if git diff -- . ':(exclude)*REVISION' | grep -vE "^(@@|diff|index|\-\-\-|\+\+\+|@generated SignedSource)" | grep "^[+-]" > /dev/null; then
```

Comment:
Can you add a comment above explaining what its looking for an example of generated header to be matched. Otherwise its going to be rough for the next person who has to adjust this.

Reply:
Done